docs/Available-Symbols-Generators.md

```diff
@@ -22,7 +33,7 @@ Most of the macros need to be configured via parameters that let you select the
 },
 ```
 
-There are several built in generators for computing parameter values. This table summarizes those.
+There are several built in generators for computing generated symbols values:
```

Comment:
built-in

Reply:
Available built-in generators for computing generated symbols values are tabulated below.